modules/ecs__backdoor_task_def/main.py

```diff
@@ -58,158 +58,151 @@
 # an @ symbol to separate the data and its region; for example:
 #     --instance-ids 123@us-west-1,54252@us-east-1,9999@ap-south-1
 # Make sure to add all arguments to module_info['arguments_to_autocomplete']
-parser.add_argument('--task-definition',required=False,default=None,help='A task definition ARN')
-parser.add_argument('--cluster',required=False,default=None,help='Cluster ARN to host task')
-parser.add_argument('--lhost',required=False,default=None,help='IP/domain to catch credentials via POST')
-parser.add_argument('--lport',required=False,default=80,help='Port to catch creds, defaults to 80')
-parser.add_argument('--execution-role',required=False,default=None,help='ARN of task role, defaults to what is provided in the task definition')
```

Comment:
Task role and execution role are different, changed this to task-role to avoid confusing them